sdk/translation/Azure.AI.Translation.Document/tests/TranslationOperationLiveTests.cs

```diff
@@ -19,7 +19,7 @@ public class TranslationOperationLiveTests : DocumentTranslationLiveTestBase
         /// </summary>
         /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
         public TranslationOperationLiveTests(bool isAsync)
-            : base(isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
```

Comment:
revert

Reply:
Are you sure?  This is left in all over the code to make it easy to swap back and forth between record and playback.  Happy to revert either way.

Reply:
I would just leave it as it was - but no big deal either way.